src/bci_build/package/appcontainers.py

```diff
@@ -271,6 +271,80 @@
 ]
 
 
+def _get_postfix_kwargs(os_version: OsVersion):
+    kwargs = {
+        "os_version": os_version,
+        "is_latest": os_version in CAN_BE_LATEST_OS_VERSION,
+        "version": "%%postfix_version%%",
+        "version_in_uid": False,
+        "replacements_via_service": [
+            Replacement(
+                regex_in_build_description="%%postfix_version%%",
+                package_name="postfix",
+                parse_version="minor",
+            )
+        ],
+        "package_list": [
+            "bash",
+            "postfix",
+            "postfix-ldap",
+            "postfix-lmdb",
+            "mandoc",
+            "netcfg",
+            "cyrus-sasl",
+            "cyrus-sasl-plain",
+            "ca-certificates",
+            "ca-certificates-mozilla",
+            "spamassassin-spamc",
+            "spamass-milter",
+            "timezone",
+            "-busybox-man",
+            "-busybox-sendmail",
+            "-busybox-iproute2",
```

Comment:
You do not have to explicitly remove busybox related packages from the container, as you are using the default base image and not busybox. This should also fix the build failures

Reply:
addressed